source/guides/references/migration-guide.md

```diff
@@ -93,6 +93,92 @@ it('allows user to login', {
 })
 ```
 
+## Module API results
+
+To more accurately reflect result data for runs with test retries, the structure of each run's `tests` object resolved from the `Promise` returned from `cypress.run()` of the Module API has changed.
+
+Mainly there is a new `attempts` Array on each `test` which will reflect the result of each test retry.
+
+{% badge danger Before %} `results.run.tests`  Module API results
+
+```json
+{
+  // ...
+  "runs": [{
+    // ...
+    "screenshots": [{
+      "screenshotId": "8ddmk",
+      "name": null,
+      "testId": "r2",
+      "takenAt": "2020-08-05T08:52:20.432Z",
+      "path": "User/janelane/my-app/cypress/screenshots/spec.js/test (failed).png",
+      "height": 720,
+      "width": 1280
+    }],
+    "tests": [{
+      "testId": "r2",
+      "title": [ "test" ],
+      "state": "failed",
+      "body": "function () {\n  expect(true).to.be["false"];\n}",
+      "stack": "AssertionError: expected true to be false\n' +
+        '    at Context.eval (...cypress/integration/spec.js:5:21",
+      "error": "expected true to be false",
+      "timings": {
+        "lifecycle": 16,
+        "test": {...}
+      },
+      "failedFromHookId": null,
+      "wallClockStartedAt": "2020-08-05T08:38:37.589Z",
+      "wallClockDuration": 1171,
+      "videoTimestamp": 4486
+    }],
+  }],
+  // ...
+}
+```
+
+{% badge success After %} `results.run.tests` Module API results
+
+```json
+{
+  // ...
+  "runs": [{
+    // ...
+    "tests": [
+      {
+        "attempts": [{
```

Comment:
could you order these keys similar to the typedefs?

Reply:
I have a nitpick for things being alphabetical. Can we have our options in app be  alphabetical 😅 But yah, I will relook at the definitions.